### PR TITLE
Amendments to Intro workshop

### PR DESCRIPTION
**Pull request by ameliejyc**
Added learning outcomes to each page within the Intro workshop.
Amended wording where appropriate.
Added in action points (relates #21)

### CODE REVIEW

File: 03-xmlhttprequest.md [hunk 1]
```diff
@@ -42,3 +48,11 @@ Following is an explanation of each line of code from the block above.
   ii. *url*: The requested URL (in this case a local file path: "xmlhttp_info.txt") - in many cases this will be the URL of the server you are querying.  
   iii. *async* (optional): whether the call should be asynchronous or not. false means it waits for a response from the server before continuing execution of the code. The default value is true, which allows you to execute other scripts while waiting for the response. This is generally preferable.
 6. `xhr.send();` --- this method sends the request to the server. Use this after setting up the XHR with the .open() method. If you are GETting, it takes no parameter, but if you are POSTing, it may take a parameter of the string you wish to post.
+
+### An aside about AJAX requests
+
+AJAX stands for Asynchronous JavaScript And XML.
+
+And AJAX call is an asynchronous request initiated by the browser which does not result in the page having to reload. AJAX requests are sometimes called XHR requests because this is the name most browsers give to the object used to send an AJAX request.
```
Comment (mantagen) [inline 03-xmlhttprequest.md]: 'An AJAX call'

**Comment by ameliejyc** [comment]
Hey @mantagen we've made both of those changes :)

File: 02-http.md [hunk 2]
```diff
@@ -15,21 +22,34 @@ HTTPS and FTPS are secure versions of HTTP and FTP. They use SSL (Secure Sockets
...
 #### POST
 
-The `POST` method is used to send data to the server. The resource at the path specified in the request will deal with the data. The data is sent in the body of the request with the encoding method specified in an additional header called `Content-Type` so that the server knows how to decode the information. A `Content-Length` header is also required so that the server knows when it has received all of the information (a large amount of data might be split across multiple requests).
+* Sends data to the server.
+* This data (e.g., from an HTML form) is processed to a specified path to modify or update a resource.
+* The data values are sent in the body of the request, in the format that the `Content-Type` header specifies. This way the server knows how to decode the information.
+* A `Content-Length` header is also required so that the server knows when it has received all of the information (a large amount of data might be split across multiple requests).
+* Unlike `GET`, `POST` data is not displayed in the url.
+
+EXAMPLE: Submitting a comment on a message board.
```
Comment (Heathercoraje) [inline 02-http.md]: I really like how you guys put these examples of each topic, it makes way easier to understand the topics

File: mentor-notes.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+## Notes for mentors
+
+### Structure of Intro Workshop
+
+This is a 1 hour and 45 minute workshop with quite a lot of reading material to cover. As is says in the readme, students should spend about 10 minutes reading each page and it will be followed by 10-15 minutes of questions and discussion. Note that the asynchronous section would benefit from going through the Loupe link on the projector.
```
Comment (Heathercoraje) [inline mentor-notes.md]: as **it** says in the readme?

File: 02-http.md (hunk 2 above)
Comment (ameliejyc) [reply 02-http.md]: Thanks Heather! ❤️

**Comment by mantagen** [comment]
@lucymonie I see you've only just been assigned to this so it might be slightly out of the blue (apologies if that is the case).

This PR comes as part of curriculum planning based on [cohort feedback on API week](https://github.com/foundersandcoders/master-reference/issues/630).

The week will start in London very soon and the deadline for this to go in has actually already passed (eek), so if you could check it out ASAP would be muchos appreciados!

**Pull request merged**